package.json

```diff
@@ -8,7 +8,8 @@
     "fix": "lerna run fix",
     "postinstall": "yarn run bootstrap",
     "compile": "lerna run compile",
-    "test": "lerna run test",
+    "test:node": "lerna run test:node",
```

Comment:
There should still be a `test` script, as it is expected that `yarn test` will run the tests. It could either default to just Node, or run both `test:node` and `test:browser`.

Reply:
Done, also changed to `nyc` and added node8 job.